File: c/normal_sum.c
#include <stdio.h>
#include <time.h>
const long int NUM = 10000;

long int calcSum(long int num) {
  long int sum = 0;

  for (long int i = 1; i <= num; i++)
  {
    sum = sum + i;
  }

  return sum;
}

int main()
{
  clock_t start, end;
  start = clock();
  long int result = 0;

  for (long int i = 1; i <= NUM; i++)
  {
    result = result + calcSum(i);
  }

  
  printf("NUM: %ld, result: %ld \n", NUM, result);
  end = clock();
  printf( "time: %lu ms", end - start);
  return 0;
}

File: js/normal_sum.js
const NUM = 10000;

const calcSum = (num) => {
  let sum = 0;
  for (let i = 1; i <= num; i++) {
    sum = sum + i;
  }
  return sum;
}

function main() {
  let result = 0;

  for (let i = 1; i <= NUM; i++) {
    result = result + calcSum(i);
  }

  console.log(`NUM: ${NUM}, result: ${result}`);
}

console.time('time');
main();
console.timeEnd('time');
